### PR TITLE
*: clean up some code

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### What problem does this PR solve?

Minor clean up the code.

### What is changed and how it works?

1. remove some unused functions
2. remove the hot spot statistic after the store becomes tombstone.

### Check List

Tests

- Unit test
-

### CODE REVIEW

File: server/statistics/store.go [hunk 1]
```diff
@@ -111,24 +88,24 @@ func (s *StoresStats) GetStoresLoads() map[uint64][]float64 {
...
-	return store.IsTombstone() || store.IsUnhealthy() || store.IsPhysicallyDestroyed()
-}
-
 // FilterUnhealthyStore filter unhealthy store
 func (s *StoresStats) FilterUnhealthyStore(cluster core.StoreSetInformer) {
 	s.Lock()
 	defer s.Unlock()
 	for storeID := range s.rollingStoresStats {
-		if s.storeIsUnhealthy(cluster, storeID) {
+		if storeIsUnhealthy(cluster, storeID) {
```
[the remark this answers is not in the document]
Comment (rleungx) [reply server/statistics/store.go:96]: I think it's better to use a function, we may use it in the future.

[the remark this answers is not in the document]
Comment (rleungx) [reply server/statistics/store.go:96]: make sense

**Review by lhy1024 — APPROVED** [review]

**Review by JmPotato — APPROVED** [review]

**Pull request merged**